source/schemas/enforce-a-schema.txt

```diff
@@ -40,38 +37,41 @@ Define a Schema
 ---------------
 
 You can define a schema for a collection in the App Services UI or with the
-App Services CLI. 
+App Services CLI.
 
-When using the App Services UI, you can choose to: 
+When using the App Services UI, you can choose to:
 
 - Generate a schema from existing data in the collection, and modify as needed.
 - Define a schema manually by adding field-level schema definitions.
 
 .. tabs-realm-admin-interfaces::
-   
+
    .. tab::
       :tabid: ui
-      
+
       .. procedure::
 
-         .. step:: Navigate to the Collection Schema Screen
+         .. step:: Navigate to the Schema Screen
 
-            In the left navigation menu, click :guilabel:`Schema` beneath :guilabel:`Data
-            Access` to open the schema editor. Atlas App Services scans your linked cluster for
-            existing collections and lists them on the left side of the schema
-            editor.
+            In the left navigation menu, click :guilabel:`Schema` beneath
+            :guilabel:`Data Access` to open the schema editor. The Collections
+            configuration screen displays in the :guilabel:`Collections` tab.
+            Atlas App Services scans your linked cluster for existing
+            collections and lists them on the left side of the schema editor.
 
-            Find and select your collection from the menu to open the collection
-            configuration screen.
+            Find and select your collection from the menu to display the
```

Comment:
_display_ is a tricky verb. You've used it correctly, but it always reads as jargon-y to me. Something like _show_ would work just as well here, I think. Our style guide doesn't say to avoid _display_ entirely, so totally up to you.

Reply:
I totally get it (_appear_ makes me prickly; our style guide doesn't mind that one either. I'll update